agent/commands/setup.go

```diff
@@ -15,7 +15,7 @@
 package commands
 
 import (
-	"fmt"
+	fmtp "fmt"
```

Comment:
Instead we can apply `nolint` to the whole file. Renaming std package just to bypass linter is not best idea.

Reply:
[Done](https://github.com/percona/pmm/pull/2302/files#diff-10b7abc985dfc8f94129edff8605bf58f6264ebf6f2bf30a0f5e180827c9bb2dR15)